packages/app/pages/[modelName]/[documentTitle]/index.tsx

```diff
@@ -79,6 +87,53 @@ const EditDocumentPage = ({
         refreshDataInPlace(router)
     }, [formData.state, refreshDataInPlace])
 
+    useEffect(() => {
+        const privileges = user.privilegesForModelAndWorkflowNode(
+            modelName,
+            model.workflow.currentNode
+        )
+        // @ts-expect-error
```

Comment:
Apparently `"dom", "dom.iterable"` in our `tsconfig.json` file doesn't include this browser API. The `userActivation` feature became [baseline](https://developer.mozilla.org/en-US/docs/Web/API/UserActivation) in 2023.